src/services/print.js

```diff
@@ -710,7 +710,7 @@ ngeo.Print.prototype.encodeVectorStyleStroke_ = function(symbolizer, strokeStyle
   var strokeColor = strokeStyle.getColor();
   if (strokeColor !== null) {
     var strokeColorRgba = ol.color.asArray(strokeColor);
-    symbolizer.strokeColor = goog.color.rgbArrayToHex(strokeColorRgba);
+    symbolizer.strokeColor = goog.color.rgbArrayToHex(/** @type {Array.<number>} */ (strokeColorRgba));
```

Comment:
Instead of a cast, what about using an instanceof `goog.asserts.assertInstanceof(strokeColorRgba, Array)`?